artichoke-backend/src/extn/core/string/mod.rs

```diff
@@ -177,4 +177,18 @@ mod tests {
         let result = interp.eval(test.as_bytes());
         unwrap_or_panic_with_backtrace(&mut interp, SUBJECT, result);
     }
+
+    #[test]
```

Comment:
```suggestion
    #[test]
    #[cfg(feature = "core-regexp")]
```

This will ensure this test only runs if `Regexp` support is enabled.